backend/src/main/java/wooteco/retrospective/presentation/member/MemberController.java

```diff
@@ -26,6 +26,13 @@ public MemberController(MemberService memberService) {
     public ResponseEntity<MemberLoginResponse> loginMember(@Valid @RequestBody MemberLoginRequest request) {
         MemberLoginDto requestDto = new MemberLoginDto(request.getName());
         MemberLoginResponse response = new MemberLoginResponse(memberService.loginMember(requestDto).getToken());
+
         return ResponseEntity.ok(response);
```

Comment:
```suggestion
        return ResponseEntity.ok(
                new MemberLoginResponse(token)
        );
```

그러면 이쪽도 깔끔하게 inline으로 처리 가능하겠네요